router/internal/cdn/persisted_operation.go

```diff
@@ -224,3 +224,7 @@ func NewPersistentOperationClient(endpoint string, token string, opts Persistent
 		},
 	}, nil
 }
+
+func (cdn *PersistentOperationClient) Close() {
```

Comment:
Can you rename it to "PersistedOperationClient"?
This name is just wrong.. (I know you didn't chose it)

Reply:
Done